packages/nectar/factory-girl.patch

```diff
@@ -0,0 +1,11 @@
+--- node_modules/factory-girl/package.json	2022-02-09 20:00:02.000000000 +0300
++++ node_modules/factory-girl/package.backup.json	2022-02-09 20:06:43.000000000 +0300
+@@ -51,7 +51,7 @@
+       "url": "http://github.com/aexmachina/factory-girl/raw/master/LICENSE"
+     }
+   ],
+-  "main": "./src/index",
++  "main": "./index",
```

Comment:
please submit PR in https://github.com/simonexmachina/factory-girl with this change

Reply:
https://github.com/simonexmachina/factory-girl/pull/156